components/api/core/api.bal

```diff
@@ -35,7 +36,10 @@ http:ServiceEndpointConfiguration celleryHubAPIEPConfig = {
     ]
 };
 
-listener http:Listener ep = new(9090, config = celleryHubAPIEPConfig);
+filter:CaptchaRequestFilter catpchaFilter = new;
+filter:validateRequestFilter validatorFilter = new;
+
+listener http:Listener ep = new(9090, config = { filters: [validatorFilter, catpchaFilter]});
```

Comment:
This is not a validator filter. This is an authentication filter. Can we rename this ?